gamehendge/templates/gamehendge/map.html

```diff
@@ -1,38 +1,167 @@
 <html>
 <head>
 {% load static %}
-	<link rel="stylesheet" type="text/css" href="{% static 'gamehendge/styling.css' %}">
+    
+    <!-- CSS
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+    <link rel="stylesheet" type="text/css" href="{% static 'gamehendge/styling.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static 'gamehendge/skeleton.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static 'gamehendge/normalize.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static 'gamehendge/style.css' %}">
+
 	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    
+
+    
+    <!-- Mobile Specific Metas
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <!-- FONT
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <link href="//fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">
+
+  <!-- Mobile Specific Metas
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
 	<title>Divided sky, the wind blows high!</title>
+    
 </head>
 <body>
-	<h3>Divided Sky v0.0.0</h3>
```

Comment:
I was worried this was redesigned for only mobile phones. That's not the case. That's good.